parent/pom.xml

```diff
@@ -235,8 +235,8 @@
         <cxf.woodstox.core.version>6.6.0</cxf.woodstox.core.version>
         <cxf.woodstox.stax2-api.version>4.2.1</cxf.woodstox.stax2-api.version>
         <cxf.wsdl4j.version>1.6.3</cxf.wsdl4j.version>
-        <cxf.jakarta.wss4j.version>3.0.4-SNAPSHOT</cxf.jakarta.wss4j.version>
-        <cxf.santuario.version>3.0.5-SNAPSHOT</cxf.santuario.version>
+        <cxf.jakarta.wss4j.version>3.0.3</cxf.jakarta.wss4j.version>
+        <cxf.santuario.version>3.0.4</cxf.santuario.version>
```

Comment:
@coheigea we could remove `santuario`, the `wss4j` references `3.0.4-SNAPSHOT` that did not exist for some reasons, I had to manually add the dependency override for it

Reply:
OK done, thanks.